src/main/java/executor/command/CommandConvert.java

```diff
@@ -56,11 +57,12 @@ public void execute(TaskList taskList) {
      * @return the amount which user wishes to convert is returned
      */
     private Double extractAmount(CommandType commandType, String userInput) {
-        String amountStr = Parser.parseForPrimaryInput(commandType, userInput);
         try {
+            String amountStr = Parser.parseForPrimaryInput(commandType, userInput);
```

Comment:
This won't result in an error so it shouldn't be in the try-catch block.

Reply:
Actually it helps to prompt the user to enter an amount. Otherwise it just prints the default error message which is to follow the format given. However, i will remove that regardless as i have a feeling if i don't the code quality might be affected.

Reply:
Managed to refactor this already. So amountStr wont be enclosed in try catch unnecessarily